### ISSUE TITLE
Django 4 beta 1: Exception 'never_cache didn't receive an HttpRequest'

### ISSUE THREAD

**Issue by hleroy** [codingjoe/django-health-check#311]
While testing a project with Django 4 beta 1, I received the following error:

```
Exception Type: TypeError at /ht/
Exception Value: never_cache didn't receive an HttpRequest. If you are decorating a classmethod, be sure to use @method_decorator.
```

**Any idea what could cause it?** 

Configuration and traceback below:

```
Request Method: GET
Request URL: https://<redacted_mysite.com>/ht/

Django Version: 4 beta1
Python Version: 3.9.7
Installed Applications:
['health_check',
 'health_check.db',
 'health_check.cache',
 'health_check.storage',
 'health_check.contrib.migrations',
 'health_check.contrib.celery',
 'health_check.contrib.redis',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django.forms',
 'widget_tweaks',
 'debug_toolbar',
 'rest_framework',
 'rest_framework.authtoken',
 'csp_helpers',]
Installed Middleware:
['debug_toolbar.middleware.DebugToolbarMiddleware',
 'django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'csp.middleware.CSPMiddleware']

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/views/generic/base.py", line 101, in dispatch
    return handler(request, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/views/decorators/cache.py", line 52, in _wrapped_view_func
    raise TypeError(

Exception Type: TypeError at /ht/
Exception Value: never_cache didn't receive an HttpRequest. If you are decorating a classmethod, be sure to use @method_decorator.
```

**Comment by erdos4d** [comment]
I have this exact problem and pull request #314 fixes it for me.

**Comment by gdalmau** [comment]
@hleroy @KristianOellegaard This can be closed, fixed in #314
